sklearn/tests/test_common.py

```diff
@@ -57,13 +56,18 @@
     check_transformer_data_not_an_array,
     check_transformer_n_iter,
     check_non_transformer_estimators_n_iter,
-    CROSS_DECOMPOSITION)
+    CROSS_DECOMPOSITION,
+    check_estimators_pfit_general,
+    check_estimators_pfit_n_features,
+    check_estimators_pfit_reset_when_fit,
+    check_estimators_pfit_effect,
+    check_estimators_pfit_classes_arg)
 
 
 def test_all_estimator_no_base_class():
     # test that all_estimators doesn't find abstract classes.
     for name, Estimator in all_estimators():
-        msg = ("Base estimators such as {0} should not be included"
+        msg = ("Base estimator such as {0} should not be included"
```

Comment:
either keep the ``s`` or remove the ``such as``. The current version doesn't seem to be gramatically correct.